packages/import/src/importers/shortcutCsv/ShortcutCsvImporter.ts

```diff
@@ -39,13 +38,13 @@ interface ShortcutIssueType {
 
 const parseBooleanColumn = (item: string) => item == "TRUE";
 const parseStringArrayColumn = (item: string) => item.split(";").filter(s => s.length > 0);
-const parseInt = (item: string) => Number.parseInt(item) || 0;
+const parseInt = (item: string) => (item ? Number.parseInt(item) : undefined);
 const parseDate = (item: string, _: any, __: any, row: string[]) => {
   if (item.length <= 0) {
     return null;
   }
   // Inoptimal method for finding the timezone UTC offset, we parse it from the UTC offset column in this row each time
-  const utcOffset = row.find(c => /[+-]([01]\d|2[0-4])(:?[0-5]\d)?/g.test(c)) || "";
+  const utcOffset = row.find(c => /^[+-]([01]\d|2[0-4])(:?[0-5]\d)?$/g.test(c)) || "";
```

Comment:
Found an issue in the Shortcut importer with data specific to the CSV I tested with. The RegExp is too broad and would catch what looks like an UTC offset but is was just part of a uuid 😬 . Updated to only match when the value takes the entire column